deploy_local_testnet.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -i
```

Comment:
Why not use `#!/usr/bin/env bash` for sure? In that case it could work on any *nix with non-standard bash location.